config/i18n-tasks.yml

```diff
@@ -102,8 +102,8 @@ ignore_unused:
   - 'date.formats.*'
   - 'general.*'
   - 'errors.*'
+  - '{devise,kaminari,will_paginate}.*'
```

Comment:
Non-blocking: I'd love to see the `devise` line not added to this `ignore_unused` block.